docs/pywbemcli/commands.rst

```diff
@@ -1556,6 +1560,57 @@ Example:
 
     $ pywbemcli --name myserver server remove-mof mymodel.mof
 
+See :ref:`pywbemcli server remove-mof --help` for the exact help output of the
+command.
+
+.. _`Server schema command`:
+
+``server schema`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``server schema`` command returns information on the :ref:`DMTF Schema`
+defined for classes in the namespace()s) of the WBEM server of the
+:term:`current connection`.  It provides information on the maximum value class
+version qualifier (which may represent the DMTF ) in the namespace, whether the
```

Comment:
"maximum value class version qualifier (which may represent the DMTF)" ???
Also, we should make it clear that each namespace can have multiple schemas.

How about:
"For each schema in each namespace, it provides information on the schema version, whether the schema has experimental classes, and the number of classes in the schema."

Reply:
DONE. 

However, we are still using schema in two different contexts, CIM Schema (prefix chars on class) and what DMTF releases.

Reply:
On "CIM schema": At one point, we tried to establish in the DMTF documents that "CIM schema" is the general construct and "CIM Schema" the one from DMTF. This even made it into glossaries.

However, I think this is too subtle and does not work in practice.

Therefore, we could for example agree that in pywbem projects, we use "CIM schema" for the general construct and "DMTF CIM Schema" for the one from DMTF.

Reply:
Not a problem